e2e/react-start/css-inline/src/routes/__root.tsx

```
@@ -0,0 +1,30 @@
+/// <reference types="vite/client" />
+import { HeadContent, Scripts, createRootRoute } from '@tanstack/react-router'
+import { Shell } from '~/components/Shell'
+import type { ReactNode } from 'react'
```

ESLint

[error] `react` type import should occur before import of `~/components/Shell`

(import/order)